libraries/AP_InertialSensor/AP_InertialSensor_LSM9DS1.cpp

```diff
@@ -0,0 +1,525 @@
+/*
+ *  This program is free software
+*/
+#include <AP_HAL/AP_HAL.h>
+#if CONFIG_HAL_BOARD == HAL_BOARD_LINUX
```

Comment:
Why?

Reply:
Hi
Thank you for finding a mistake. I've already fixed it.